### PR TITLE
Convert to jekyll site

### PR DESCRIPTION
**Pull request by stianjensen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
 # dmpro2014.github.io
+
+## Setup
+
+**Prerequisites:** ruby, rubygems and bundler.
+
+When these are installed, you can run `bundle install`.
```
Comment (cristeahub) [inline README.md:7]: What ruby version?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+_site
+.*
```
Comment (cristeahub) [inline .gitignore:2]: I suggest `*.sw*` for vim files too.

File: README.md (hunk 1 above)
Comment (stianjensen) [inline README.md:7]: I don't think it matters, but I use 2.0.x

Comment (cristeahub) [inline README.md:7]: I suggest we add `ruby version 2.x.x` in the readme

**Comment by cristeahub** [comment]
Isn't jekyll sited built using markdown?

**Comment by stianjensen** [comment]
You may use markdown if you want, but html is also possible.

Jekyll basically copies all files directly into the _site folder, which means html, css, js etc. are automatically supported

File: README.md (hunk 1 above)
Comment (stianjensen) [inline README.md:7]: Jekyll 2 requires ruby 1.9.3 or higher, according to http://jekyllrb.com/docs/installation/

File: .gitignore (hunk 2 above)
Comment (stianjensen) [inline .gitignore:2]: They should already be covered by .*

**Comment by cristeahub** [comment]
Ok, shouldn't we move the index.html file to the _site folder?

**Comment by stianjensen** [comment]
> Ok, shouldn't we move the index.html file to the _site folder?

No. The `_site` folder is automatically built and overwritten by jekyll.

Jekyll looks for files in the root directory and sub-directories. All files that are not marked by the `exclude` property in the config will then be copied into the `_site` folder. Another exception is folders that start their name with an underscore, which typically have a special meaning in jekyll (`_sass`, `_data`, etc.)

File: .gitignore (hunk 2 above)
Comment (cristeahub) [inline .gitignore:2]: Really? isn't that just for files that start with .? so `markdown.md.swp` wouldn't be covered by that? If so the `.*` would ignore all files :P

Comment (stianjensen) [inline .gitignore:2]: No, I mean that swp files always start with a dot

File: README.md (hunk 1 above)
Comment (cristeahub) [inline README.md:7]: Ok, I would suggest adding that to the readme, but it's not crucial.

**Comment by cristeahub** [comment]
Ok, cool :+1:

**Comment by stianjensen** [comment]
ping (added ruby version)

File: .gitignore (hunk 2 above)
Comment (cristeahub) [inline .gitignore:2]: Yes of course, I'm not sure where my head was here.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.PHONY: all run build
```
Comment (cristeahub) [inline Makefile:1]: I usually like to organize my makefiles like this:

```
.PHONY: all
all:
    command

.PHONY: run
run:
    command

etc...
```

This is personal preference and not semantically changing anything, so I'll merge this regardless, but thought I'ld share my preferences ^^

Comment (stianjensen) [inline Makefile:1]: Fix it in a pull request \o/

**Pull request merged**